simx/src/messages/TranslationProvider.tsx

```diff
@@ -6,6 +6,13 @@
 import { ReactNode } from "react";
 import { IntlProvider, MessageFormatElement } from "react-intl";
 import enMessages from "./ui.en.json";
+import esesMessages from "./ui.es-es.json";
+import jaMessages from "./ui.ja.json";
+import koMessages from "./ui.ko.json";
+import nlMessages from "./ui.nl.json";
+import plMessages from "./ui.pl.json";
+import ptbrMessages from "./ui.pt-br.json";
+import zhtwMessages from "./ui.zh-tw.json";
```

Comment:
Interesting. We normally dynamically load the messages but as they're so small for the sim this absolutely makes sense.